lua/pug/modules/lagdetection.lua

```diff
@@ -95,28 +109,35 @@ u.addHook("Tick", "LagDetection", function()
 
 	if not inTolerance then
 		skips = skips + 1
-		if ( lag.rate > lag.trigger ) or ( skips > lag.skips ) then
-			ServerLog( "LAG: ", lag.rate, "Th: ", lag.threshold )
-			ServerLog( "Average: ", sample.mean, "Skips: ",
-			skips, "/", lag.skips )
+		print("Skips: " .. tostring(skips))
+		print("Rate: " .. tostring(lag.rate))
+		print("Average: " .. tostring(sample.mean))
+
+		-- IF there's casual lag
+		if ( sample.mean <= lag.trigger ) or ( skips >= lag.skips ) then
```

Comment:
The previous line was:
```lua
if ( lag.rate > lag.trigger ) or ( skips > lag.skips ) then
```

Which didn't make sense to me for a couple of reasons.
Firstly, why would we want to run the cleanup functions if the tickrate is _higher_ than our trigger value?
Second, even if we swapped the operators, we'd still end up running the cleanup functions on the 1-off low tickrate events.

So instead, we changed the first check to compare the `sample.mean` against the trigger value (I suspect this was the original intention anyway).

We also changed the second check from a `>` to a `>=`

Reply:
The tolerance check should take care of 1-off low tick rate events.